docs/docs/install/docker-compose.mdx

```diff
@@ -93,20 +97,36 @@ For more information on how to use the application, please refer to the [Post In
 Downloading container images might require you to authenticate to the GitHub Container Registry ([steps here][container-auth]).
 :::
 
-### Step 4 - Upgrading
+## Next Steps
+
+### Setting Up Optional Features
+
+You can set up the following now:
+
+- [External Libraries](/docs/features/libraries.md)
+- [Hardware Transcoding](/docs/features/hardware-transcoding.md)
+- [Hardware-Accelerated Machine Learning](/docs/features/ml-hardware-acceleration.md)
+
+### Upgrading
 
 :::danger Breaking Changes
 It is important to follow breaking updates to avoid problems. You can see versions that had breaking changes [here][breaking].
 :::
 
 If `IMMICH_VERSION` is set, it will need to be updated to the latest or desired version.
 
-When a new version of Immich is [released][releases], the application can be upgraded with the following commands, run in the directory with the `docker-compose.yml` file:
+When a new version of Immich is [released][releases], the application can be upgraded and restarted with the following commands, run in the directory with the `docker-compose.yml` file:
 
-```bash title="Upgrade Immich"
+```bash title="Upgrade and restart Immich"
 docker compose pull && docker compose up -d
 ```
 
+To clean up disk space, the old version's obsolete containers can be deleted with the following command, which deletes all stopped containers. As such, make sure to start Immich (with the previous command) before running it.
```

Comment:
```suggestion
To clean up disk space, the old version's obsolete containers can be deleted with the following command, which deletes all unused container images. As such, make sure to start Immich (with the previous command) before running it.
```

Reply:
Wasn’t aware of images vs containers but looks good, will merge.

Reply:
I believe this command can be run with the containers stopped so I deleted that second sentence as well.

Reply:
actually i tried running `docker image prune` with the containers stopped and i feel like `docker compose up -d` then redownloaded some stuff? unsure, needs checking